internal/auth/auth.go

```diff
@@ -103,7 +119,7 @@ func Logout() error {
 
 func testAuth(apiKey string) error {
 	// TODO eventually replace with a call from the SDK
-	u, err := url.JoinPath(apiURL, "v1/auth/validate")
+	u, err := url.JoinPath(GetServerURL(), "v1/auth/validate")
```

Comment:
Does app.speakeasy.dev work for the API?

Reply:
These were previously two different urls as I believed our web apps were hosted on different subdomains to the api?

Reply:
It does -- path based routing